authorizer-app-backend/src/main/java/org/radarbase/authorizer/resources/SourceClientResource.kt

```diff
@@ -120,13 +119,13 @@ class SourceClientResource(
 
     @POST
     @Path("{type}/deregister")
-    fun reportDeregistration(@PathParam("type") sourceType: String, params: DeregistrationParams): Response {
-        val user = userRepository.findByExternalId(params.userId, sourceType)
-        if (user != null) {
-            if (user.accessToken == params.userAccessToken)
+    fun reportDeregistration(@PathParam("type") sourceType: String, body: DeregistrationsDTO): Response {
+        body.deregistrations.forEach { it ->
+            val user = userRepository.findByExternalId(it.userId, sourceType)
+            if (user != null && user.accessToken == it.userAccessToken)
```

Comment:
Right, at least it's consistent in the format and sending data for multiple users 😆